### PR TITLE
web: converted css.js into typescript

### PR DESCRIPTION
**Pull request by Abdulrehman-Jafer**

### CODE REVIEW

File: apps/web/src/utils/css.ts [hunk 1]
```diff
@@ -17,14 +17,14 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-export function removeCss(id) {
-  var link = document.getElementById(id);
-  link.remove();
+export function removeCss(id: string) {
+  const link = document.getElementById(id);
+  link ? link.remove() : undefined;
```
Comment (thecodrr) [inline apps/web/src/utils/css.ts:22]: A simple if statement will suffice here.

File: apps/web/src/utils/css.ts [hunk 2]
```diff
@@ -34,32 +34,33 @@ export function injectCssSrc(id, src) {
...
   if (variableCss) {
     head.removeChild(variableCss);
   }
-  let css = document.createElement("style");
+  const css = document.createElement("style");
   css.type = "text/css";
   css.id = "variables";
   // Support for IE
-  if (css.styleSheet) css.styleSheet.cssText = rule;
+  if ((css as any).styleSheet) (css as any).styleSheet.cssText = rule;
```
Comment (thecodrr) [inline apps/web/src/utils/css.ts:47]: We don't support IE so you can remove this.

**Review by thecodrr — CHANGES_REQUESTED** [review]

File: apps/web/src/utils/css.ts (hunk 2 above)
Comment (Abdulrehman-Jafer) [reply apps/web/src/utils/css.ts:47]: Ok

File: apps/web/src/utils/css.ts (hunk 1 above)
Comment (thecodrr) [reply apps/web/src/utils/css.ts:22]: @Abdulrehman-Jafer

Comment (Abdulrehman-Jafer) [reply apps/web/src/utils/css.ts:22]: Ok I'll remove the ternary..

**Review by thecodrr — APPROVED** [review]

**Pull request merged**